app/src/App.css

```diff
@@ -144,6 +143,67 @@ body::-webkit-scrollbar-thumb {
   color: #2b9e91;
 }
 
+.calendar .navBar {
+  display: grid;
+  box-sizing: border-box;
+  position: relative;
+  top: 0em;
+  left: 0em;
+  margin: auto;
+  padding: 0em 0.125em;
+  width: 100%;
+  height: auto;
+
+  grid-template-columns: 1fr 3fr 1fr;
+
+  background-color: #e86f51;
+  color: ghostwhite;
+
+  text-align: center;
+}
+
+.calendar .navBar button {
+  height: auto;
+  width: minMax(auto, 80%);
+}
+
+button.prev {
+  left: 0em;
+}
+
+button.next {
+  right: 0em;
+}
+
+.calendar .navTab {
+  display: block;
+  position: relative;
+  float: left;
+  box-sizing: border-box;
+  margin: 1%;
+  height: auto;
+  width: auto;
+
+  background-color: ghostwhite;
+  border: 2px solid #f5a462;
+  border-radius: 0.75em 0.75em;
+  box-shadow: 0.25em 0.25em 0.25em rgba(39, 71, 84, 0.75);
+  filter: none;
+
+  font-size: 1rem;
+  color: #e86f51;
+}
+
+.calendar .navTab.toggled {
+  z-index: 2;
+}
+
+.calendar .navTab.notToggled {
+  background-color: gainsboro;
+}
+
+/* TIMELINE */
+
```

Comment:
Great job on the CSS styling!

Reply:
I second this I will definitely be taking a peek later on to help me with my CSS file

Reply:
I third this, your styling work always looks so nice! 👀